source/Hardpoint.cpp

```diff
@@ -135,10 +137,11 @@ bool Hardpoint::IsHoming() const
 
 
 
-// Find out if this hardpoint has an anti-missile installed.
-bool Hardpoint::IsAntiMissile() const
+// Find out if this hardpoint has a special weapon installed
+// (e.g. anti-missile, tractor beam).
+bool Hardpoint::IsSpecial() const
 {
-	return outfit && outfit->AntiMissile() > 0;
+	return outfit && (outfit->AntiMissile() > 0 || outfit->TractorBeam() > 0.);
```

Comment:
```suggestion
	return outfit && (outfit->AntiMissile() || outfit->TractorBeam());
```
wouldnt this be the same?

Reply:
AntiMissile and TractorBeam can't be below 0, so yeah.